goss-testing/scripts/check_bios_firmware_versions.sh

```diff
@@ -57,8 +57,37 @@ function set_vars() {
   fi
 
   # Allow for output of documentation when needed
+  rc=0
+
+  DL325_FW="2.46"
+  DL385_FW="2.46"
+
+  DL325_BIOS="A43"
+  DL385_BIOS="A42"
+
+  # Taken from HFP-firmware-2.0.111516-0 but leaving here for reference in this code
+  # Appropriate versions to use for Shasta 1.5.0 or HPCM as of 11/4/2021 unless otherwise indicated by your
```

Comment:
Do we want the information here for Shasta 1.5, if this is going into the csm-1.2 branch?

Reply:
Just updated this.